.github/workflows/ci-syntax-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Syntax Tests
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '.github/workflows/ci-syntax-tests.yml'
+      - '**.sublime-syntax'
+      - '**/syntax_test_*'
+      - '**.tmPreferences'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
Should be targeting the `st4` branch, not `master`.

Reply:
st4 is no working branch, which is merged back to master?

Reply:
I mean, I'd add an st3 branch to lock the st3 state or do some st3 specific fixes and go on with master.

Reply:
But currently it's set up in a way where ST4 releases are made from `st4` branch.

I believe the reason why it's not the other way around is that the main/master branch contains also non-ST-specific syntaxes so it wouldn't fully make sense to make the ST4-specific branch the main one.

Reply:
We don't need tests for ST3 version.

The `st4` branch will be the working branch for all ST4-compatible releases. I don't plan to update anything for the ST3 version of the plugin unless any significant issues crop up. You can safely target tests only for the new version and the `st4` branch.

Reply:
OK, the commit below targets push's to st4 only.